### DISCUSSION TITLE
set sub-ass-override "force" on  specific fonts

### DISCUSSION THREAD

**Discussion by v-amorim** [mpv-player/mpv#14720] [Q&A] [answered]
Is it possible to specify which font/ass style to force override with `sub-ass-override "force"`?

For example, I only want the bottom subtitle to be forced, not the chart names

![image](https://github.com/user-attachments/assets/d9697648-bef5-4a48-9ba9-6b2e7db185db)
<p align="center">Normal ↔ Forced</p>

**Comment by v-amorim** [comment]
Just learned this, using `yes` instead of `forced` and just overriding the font achieve at least something better:

```properties
sub-ass-override=yes
sub-font="Arial" # For some reason without this, it uses the default/file_defined font
sub-ass-style-overrides=FontName="Arial"
```
![image](https://github.com/user-attachments/assets/360f54cf-b30d-4688-9965-cb9c153f4975)

Also saw about `sub-ass-styles="~~\template.ass"`, but I'm guessing the ass style naming in the template would have to match the names on each video's ass names.

**Reply by Solarflame5** [reply]
Sorry for the unrelated reply, but which font are you using in the screenshot?

**Reply by v-amorim** [reply]
@Solarflame5 its "Impress BT", I love it, found it when watching One Pace

**Reply by Solarflame5** [reply]
Thank you so much, I also love how it looks.

**Comment by llyyr** [comment, marked as answer]
There's no way to do this until something like https://github.com/libass/libass/pull/570 is merged. If you know the name of the styles you want to modify then just use `sub-ass-style-overrides=StyleName.FontName="Arial"`

**Reply by v-amorim** [reply]
Fair, at least I learned the about `sub-ass-style-overrides` and it works better than forcing, thank you!

**Comment by v-amorim** [comment]
Related to what I found out, do you know why the `sub-font` is required when using `sub-ass-style-overrides=FontName`?

Does it apply to any other parameter I include?
